### ISSUE TITLE
Python 3.5 ?

### ISSUE THREAD

**Issue by Athesdrake** [GiovanniMCMXCIX/PyDiscordRPC#4]
Hey
I'm using Python 3.5 and I get this error when I run discordrpc.py :
```python
File "D:\Python\DiscordRPC\discordrpc.py", line 17
    self.sock_reader: asyncio.StreamReader = None
                    ^
SyntaxError: invalid syntax
```
Is it a new syntax in Python 3.6 ?
Do you know how can I fix it ?

**Comment by ForLoveOfCats** [comment]
Yes, the code included in this repo requires Python 3.6+. I opened a pull request 2 weeks ago with the intention of stating that in the README. Unfortunately the repo owner appears to be MIA.

As for getting it working in 3.5.... It should be possible but I do not have the expertise to tell you how. Sorry.

**Comment by GiovanniMCMXCIX** [comment]
This is only an example.
I don't plan supporting other versions than 3.6+
